editor/store/actions.js

```diff
@@ -275,6 +304,56 @@ export function autosave() {
 	};
 }
 
+/**
+ * Returns an action object used to toggle the autosave action status.
+ *
+ * @param  {boolean} isAutosaving Is the editor autosaving?
+ * @return {Object}            Action object
+ */
+export function toggleAutosave( isAutosaving ) {
+	return {
+		type: 'DOING_AUTOSAVE',
+		isAutosaving,
+	};
+}
+
+/**
+ * Returns an action to show the autosave notice.
+ *
+ * @param  {boolean} autosaveStatus Autosave status and data including a link to the autosave.
+ * @return {Object}              Action object
+ */
+export function showAutosaveNotice( autosaveStatus ) {
+	return {
+		type: 'REQUEST_AUTOSAVE_EXISTS',
```

Comment:
Action type should correspond more-or-less to action creator name. Why is `showAutosaveNotice` dispatching as `REQUEST_AUTOSAVE_EXISTS` ?

Reply:
renames actin type REQUEST_AUTOSAVE_NOTICE

Reply:
>renames actin type REQUEST_AUTOSAVE_NOTICE

Why the inconsistency between the action creator function name and the action type name?